kpatch-build/kpatch-build

```diff
@@ -602,6 +602,13 @@ echo "Building patch module: kpatch-$PATCHNAME.ko"
 cp "$OBJDIR/.config" "$SRCDIR"
 cd "$SRCDIR"
 make prepare >> "$LOGFILE" 2>&1 || die
+
+if [[ $DISTRO == ubuntu ]]; then
+  # UBUNTU: add UTS_UBUNTU_RELEASE_ABI to utsrelease.h after regenerating it
+  UBUNTU_ABI=$(echo "$ARCHVERSION" | awk '{split($1, a, "-"); print a[2]}')
```

Comment:
The shelling out can be avoided here by using bash parameter expansion.  I think `UBUNTU_ABI=${ARCHVERSION#*-}` would work.  Also please use tabs instead of spaces.